### PR TITLE
add logout feature

### PR DESCRIPTION
**Pull request by sbaogang**
added logout feature, also fix some minor issues, added docker run without DB. 

@lilida @yongwen @haom-pinterest

### CODE REVIEW

File: deploy-service/teletraanservice/bin/server-no-db.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      target: stdout
+
+
+#
+# Mysql connection config. By default, we use the real mysql as the backend.
+# Note: the embeded mysql is for demo or test purpose only. Use the official Mysql
+# installs for production usage.
+#
+db:
+  type: embedded
```
Comment (sbaogang) [inline deploy-service/teletraanservice/bin/server-no-db.yaml]: This is really what I want to change - there is no better way I could find to not copy the whole server.yaml

File: deploy-service/teletraanservice/bin/server.yaml [hunk 2]
```diff
@@ -51,7 +51,7 @@ db:
   type: mysql
 
   # Mysql instance configurations
-  host: ${TELETRAAN_DB_HOST:-default localhost}
+  host: ${TELETRAAN_DB_HOST:-localhost}
```
Comment (sbaogang) [inline deploy-service/teletraanservice/bin/server.yaml]: I did some simple test, seems this is correct syntax

File: docker-compose-no-db.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+version: '2'
```
Comment (sbaogang) [inline docker-compose-no-db.yml]: again, I could not find a better way to not copy paste the docker-compose file

**Comment by sbaogang** [comment]
add also @sjoshi6

**Comment by haom-pinterest** [comment]
LGTM

File: deploy-board/deploy_board/webapp/security.py [hunk 4]
```diff
@@ -151,3 +142,25 @@ def login_authorized(request):
         return HttpResponseRedirect(data['origin_path'])
 
     return HttpResponseRedirect('/')
+
+def logout(request):
+    logger.debug("Logout!")
```
Comment (sbaogang) [inline deploy-board/deploy_board/webapp/security.py]: sounds good, will add it in my next PR.

**Pull request merged**